2017/AUG/AUG17/PALINGAM/palingam.py

```diff
@@ -0,0 +1,28 @@
+t = int(input())
+while t > 0 :
+    a = input()
+    b = input()
+    la = list()
+    lb = list()
+    la = 26*[0]
```

Comment:
Add space around `*` and repush so that it will trigger travis logs and we can know whats going wrong.